ebpf/event_test.go

```diff
@@ -23,6 +23,27 @@ var (
 	}
 )
 
+func TestBeautifyKey(t *testing.T) {
+	buf := &bytes.Buffer{}
+	for _, c := range []ConnectionStats{
+		testConn,
+		ConnectionStats{
+			Pid:    345,
+			Type:   0,
+			Family: 1,
+			Source: "127.0.0.1",
+			Dest:   "192.168.0.103",
+			SPort:  4444,
+			DPort:  8888,
+		},
+	} {
+		bk, err := c.ByteKey(buf)
+		assert.NoError(t, err)
```

Comment:
nit, we should almost always use `require.NoError` so that tests fail fast.  `assert.NoError` records the failure but allows the test to continue which will probably panic